docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+# Creating a tool with an API hit
+In this guide, we will use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that pulls information from an API and find the mean, min, and max of the data.
+
+---
+## Workspace Setup
+---
+### 1. Creating a workspace
+The very first step to creating a plugin is to make a plugin workspace. We initialize a plugin workspace in an empty directory with the `sdk-workspace-init` command. Run the command and fill out the prompts, which will then start the workspace initialization process.
+
+```
```

Comment:
If you suffix this with the "language", it will render nicer, e.g. "bash", "powershell", etc. after the starting ticks on the same line

Reply:
yeah i have it for the python code snippets but since it's all custom print lines to stdout it renders kinda weird... 

i added it for now, but let me know if you think we should keep it. tried powershell, bash, and sh but it highlights random words for all of them... powershell being the best imo cause it makes the $ green hahaha